src/Services/IReferenceTokenRevoker.cs

```diff
@@ -0,0 +1,6 @@
+namespace IdentityServerConfig.Services;
+
+public interface IRevoker
```

Comment:
The typename and filename do not match so I guess the type has been renamed. But I think the original filename would be a more appropriate name. As a consumer of a service that revokes something I would probably want to know specifically what it is I am revoking.

Reply:
Yes, you're right, the type has been renamed. I did this, because when thinking ahead, we also want to revoke the Client Secret. Thus I have changed the name to be more generic, so in the future we can also make a ClientSecretRevoker that inherits IRevoker.
I will change the filename to be IRevoker

Reply:
I was indeed renamed, but it has been reverted in 23ec94e